client/src/components/SentimentAnalysis/SimilarAds.js

```diff
@@ -0,0 +1,59 @@
+import { database } from '../../firebase/firebase';
+import React, { useState, useEffect } from 'react';
+
+const SimilarAds = props => {
+
+  const COLLECTION = 'ads';  
+  const headlineSentiment = `{"score": ${props.headlineScore}, "magnitude": ${props.headlineMagnitude}}`;
+  const contentSentiment = `{"score": ${props.contentScore}, "magnitude": ${props.contentMagnitude}}`;
+
+  const [results, setResults] = useState([]);
+  let resultItems = [];
```

Comment:
We can put this locally inside the `useEffect` right before its usage - also, use `const` for `resultItems` since it doesn't reference to another object later